### PR TITLE
Consolidate README

### PR DESCRIPTION
**Pull request by ChaseHall**
Most of our information has been moved to the wiki, this just cleans up the readme.

Obviously if you want to keep some of the information I removed, feel free to re-add it or let me know.

[Preview the new readme here](https://github.com/ChaseHall/IW4M-Admin/blob/2.4-pr/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# IW4MAdmin
+# IW4MAdmin [![GitHub license](https://img.shields.io/github/license/RaidMax/IW4M-Admin)](https://github.com/RaidMax/IW4M-Admin/blob/2.4-pr/LICENSE) [![GitHub stars](https://img.shields.io/github/stars/RaidMax/IW4M-Admin)](https://github.com/RaidMax/IW4M-Admin/stargazers)
 ### Quick Start Guide
 ### Version 2.4
```
Comment (RaidMax) [inline README.md:4]: Since the readme is now just a shell, let's remove the version as it's not really needed.

File: README.md [hunk 2]
```diff
@@ -45,424 +45,7 @@ Linux
...
-The anti-cheat feature is a work in progress and as such will be constantly tweaked and may not be 100% accurate, however the goal is to deter as many cheaters as possible from IW4x.
-#### Database Storage
-By default, all **IW4MAdmin** information is stored in `Database.db`. 
-
-Should you need to reset your database, this file can simply be deleted.  
-Additionally, this file should be preserved during updates to retain client information.   
-
-Setting the `ConnectionString` and `DatabaseProvider` properties in `IW4MAdminSettings.json`  
-will allow **IW4MAdmin** to use alternate methods for database storage
+## [More information may be found in our wiki.](https://github.com/RaidMax/IW4M-Admin/wiki)
```
Comment (RaidMax) [inline README.md:51]: Let's change this verbiage to "Explore the wiki (linked) to find more information"

File: README.md (hunk 1 above)
Comment (ChaseHall) [reply README.md:4]: The version as in "Version 2.4" below quick start?

File: README.md [hunk 3]
```diff
@@ -45,424 +44,7 @@ Linux
...
-The anti-cheat feature is a work in progress and as such will be constantly tweaked and may not be 100% accurate, however the goal is to deter as many cheaters as possible from IW4x.
-#### Database Storage
-By default, all **IW4MAdmin** information is stored in `Database.db`. 
-
-Should you need to reset your database, this file can simply be deleted.  
-Additionally, this file should be preserved during updates to retain client information.   
-
-Setting the `ConnectionString` and `DatabaseProvider` properties in `IW4MAdminSettings.json`  
-will allow **IW4MAdmin** to use alternate methods for database storage
+## [Explore the wiki (linked) to find more information.](https://github.com/RaidMax/IW4M-Admin/wiki)
```
Comment (RaidMax) [inline README.md:50]: Guess I should have been more explicit, just hyperlink the "wiki" text, (linked was just indicating that that word should be hyperlinked)
`## Explore the [wiki](https://github.com/RaidMax/IW4M-Admin/wiki) to find more information.`

Comment (ChaseHall) [reply README.md:50]: derp. figured that was what you meant.

**Comment by RaidMax** [comment]
Looks good - make sure to use the squash and merge option so we just get one commit. Thanks!

**Pull request merged**